dataset_configs/english/librispeech/config.yaml

```diff
@@ -4,19 +4,21 @@ documentation: |
   
   This config can be used to prepare
   `Librispeech <https://openslr.org/12>`_
+  `Librispeech(mini) <https://openslr.org/31>`_
```

Comment:
Docs look odd-- currently there are just these two link next to each other. Please e.g. write "and" in between the links